README.md

```diff
@@ -84,18 +86,49 @@ Start command will start application on 2 ports that need to be exposed to publi
 
 `--capacity` - maximum number of nodes allowed to connect - **DEFAULT** [-1] unlimited capacity
 
-`--whitelist` - Comma separated list of node id-s, if provided only these nodes will be allowed to connect. This flag can't be used together with --whitelist-file flag, only one option for setting whitelisted nodes can be used - **DEFAULT** no nodes whitelisted
+`--whitelist` - comma separated list of node id-s, if provided only these nodes will be allowed to connect. This flag can't be used together with --whitelist-file flag, only one option for setting whitelisted nodes can be used - **DEFAULT** no nodes whitelisted
 
-`--whitelist-file` - Path to file with node id-s in each line that should be whitelisted. This flag can't be used together with --whitelist flag, only one option for setting whitelisted nodes can be used - **DEFAULT** no file used
+`--whitelist-file` - path to file with node id-s in each line that should be whitelisted. This flag can't be used together with --whitelist flag, only one option for setting whitelisted nodes can be used - **DEFAULT** no file used
 
 `--fee` - value between 0-1 representing fee percentage that loadbalancer will take - **DEFAULT** [0.1]
 
 `--selection` - type of selection that is used for selecting nodes on new request, valid values are `round-robin` and `random` - **DEFAULT** [round-robin]
 
+`--payout-interval` - automatic payout interval specified as number of days
+
+`--payout-reward` - defined reward amount that will be distributed on the payout (amount in Planck)
```

Comment:
maybe make it optional and default to send all funds. Because common scenario is that you will receive periodcally amount on wallet to be disbursted.

Reply:
Where do we send the lb percentage then?

Reply:
argh, true, bit wierd then. You have to check if you got money to be disbursed, otherwise, you gonna send your own money. We could maybe, allow entering separate address where fee would be payout. But yeah, let's make this required for now.

Reply:
We can eventually support both options, let's stick with this for now. I will open an issue where we can discuss this more and define the final solution.